src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -8,6 +8,8 @@
 * (Deliberately contains bugs.)
 */
 public class TorpedoStore {
+  //moved this here from fire() method
+  private Random generator = new Random();
```

Comment:
Seems good to me.
This way we only create one instance of the Random class, instead of creating it everytime the fire method is called.